engine/src/flutter/display_list/display_list_unittests.cc

```diff
@@ -5251,7 +5251,6 @@ TEST_F(DisplayListTest, ClipRectRRectPathPromoteToClipRect) {
   DlRoundRect clip_rrect = DlRoundRect::MakeRect(clip_rect);
   DlRect draw_rect = clip_rect.Expand(2.0f, 2.0f);
   DlPath clip_path = DlPath::MakeRoundRect(clip_rrect);
-  ASSERT_TRUE(clip_path.IsRoundRect());
```

Comment:
Rather than eliminate this, change it to `IsRect()`.

Reply:
The proposed change would not work without an updated version of Skia, which is currently blocked on these very asserts. Reading the tests, it seems the functionality is fully validated by the DL asserts.

I guess the options are

1) drop the path asserts because they don't add much value in DL tests
2) re-add updated asserts after the skia roll
3) update skia and the asserts atomically in a manual roll

Of these, #1 minimizes friction for the current & future Skia updates.

Reply:
Ah, yes, #1. The DL stuff is enough.